drake/systems/plants/collision/FCLModel.h

```diff
@@ -1,5 +1,5 @@
-#ifndef __DrakeCollisionFCLModel_H__
-#define __DrakeCollisionFCLModel_H__
+#ifndef DRAKE_COLLISION_FCL_MODEL
```

Comment:
The usual convention for header guards is to end them with "_H_" so they won't accidentally collide with real macros. (So `DRAKE_COLLISION_FCL_MODEL_H_`). Trailing underscores are allowed freely by the language although leading underscores are restricted.